### PR TITLE
feat(save-and-load-assessment): add uploading functionality to load assessment button

### PR DESCRIPTION
**Pull request by shanisebarona**
#### Description of changes

This PR adds the uploading functionality to the load assessment button and updates relevant tests. Lack of code coverage for `load-assessment-button.tsx` will be addressed in the next PR. Empty function in `assessment-store.ts` is setting up infra and will be completed in a future PR.

#### Pull request checklist

- [x] Addresses an existing issue: Feature request #653
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [n/a] (UI changes only) Added screenshots/GIFs to description above
- [n/a] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/tests/unit/tests/DetailsView/components/details-view-command-bar.test.tsx [hunk 1]
```diff
@@ -14,8 +14,9 @@ import {
 } from 'DetailsView/components/save-assessment-button';
 import {
     LoadAssessmentButton,
+    LoadAssessmentButtonDeps,
     LoadAssessmentButtonProps,
-} from 'DetailsView/components/load-assessment-button';
+} from '../../../../../DetailsView/components/load-assessment-button';
```
Comment (jalkire) [inline src/tests/unit/tests/DetailsView/components/details-view-command-bar.test.tsx:19]: Was there any particular reason for making this import path relative?

File: src/DetailsView/components/load-assessment-button.tsx [hunk 2]
```diff
@@ -3,14 +3,46 @@
...
+            <InsightsCommandButton
+                iconProps={{ iconName: 'FabricOpenFolderHorizontal' }}
+                onClick={this.getAssessmentForUpload}
+            >
                 Load assessment
             </InsightsCommandButton>
         );
     }
+
+    private getAssessmentForUpload = () => {
```
Comment (jalkire) [inline src/DetailsView/components/load-assessment-button.tsx:29]: Given that this button says "Load assessment", I'm wondering if "upload" is the right word to use in the code here and in the other places in this PR (`UploadAssessmentPayload`, `onUploadAssessment`, etc). Do you have a preference for upload?

File: src/background/stores/assessment-store.ts [hunk 3]
```diff
@@ -147,6 +149,10 @@ export class AssessmentStore extends BaseStoreImpl<AssessmentStoreData> {
         this.updateTargetTabWithId(tabId);
     };
 
+    private onUploadAssessment = (payload: UploadAssessmentPayload): void => {
+        //update state
```
Comment (jalkire) [inline src/background/stores/assessment-store.ts:153]: Dan had mentioned on a call including a ADO work item number with this comment referencing a user story or feature tracking this work. That could be helpful!

File: src/tests/unit/tests/DetailsView/components/load-assessment-button.test.tsx [hunk 4]
```diff
@@ -6,13 +6,12 @@ import { shallow } from 'enzyme';
 import {
     LoadAssessmentButton,
     LoadAssessmentButtonProps,
-} from 'DetailsView/components/load-assessment-button';
-
+} from '../../../../../DetailsView/components/load-assessment-button';
```
Comment (jalkire) [inline src/tests/unit/tests/DetailsView/components/load-assessment-button.test.tsx:9]: Same comment as earlier: should this import path be changed?

File: src/tests/unit/tests/DetailsView/components/details-view-command-bar.test.tsx (hunk 1 above)
Comment (shanisebarona) [reply src/tests/unit/tests/DetailsView/components/details-view-command-bar.test.tsx]: Sometimes the paths give me an error and it recommends this as the fix. Will switch back and make sure to run the build when switching branches!

File: src/DetailsView/components/load-assessment-button.tsx (hunk 2 above)
Comment (shanisebarona) [reply src/DetailsView/components/load-assessment-button.tsx:29]: You're right, the naming here is inconsistent, will update `upload` to `load`. Thanks for catching this!

**Review by jalkire — APPROVED** [review]
👍

**Pull request merged**